gooddata-sdk/gooddata_sdk/table.py

```diff
@@ -219,8 +230,12 @@ def _prepare_tabular_definition(
     attributes: list[Attribute], filters: list[Filter], metrics: list[Metric]
 ) -> ExecutionDefinition:
     dims = [
-        [a.local_id for a in attributes] if len(attributes) else None,
-        ["measureGroup"] if len(metrics) else None,
+        ExecTableDimension(
+            item_ids=[a.local_id for a in attributes] if attributes else None,
+        ),
+        ExecTableDimension(
+            item_ids=[_MEASURE_GROUP_IDENTIFIER] if len(metrics) else None,
+        ),
```

Comment:
Just out of curiosity. Why do conditions differ?

Reply:
Mistake on my side, they should be aligned :)